tests/localosmosis/docker-compose.yml

```diff
@@ -1,15 +1,25 @@
 version: "3"
 
 services:
+
   osmosisd:
     image: local:osmosis
-    user: "root:root"
+    build:
+      context: ../../
+      dockerfile: Dockerfile
+      args:
+        RUNNER_IMAGE: alpine:3.16
+        GO_VERSION: 1.18
     volumes:
-      - ./.osmosisd/config:/osmosis/.osmosisd/config
-      - ./.osmosisd/data:/osmosis/.osmosisd/data
-      - ./.osmosisd/wasm:/osmosis/.osmosisd/wasm
+      - ./scripts/setup.sh:/osmosis/setup.sh
+    entrypoint:
+      - /osmosis/setup.sh
     ports:
-      - "26657:26657"
-      - "1317:1317"
+      - 26657:26657
+      - 1317:1317
       - "9090:9090"
       - "9091:9091"
```

Comment:
A bit confused why lines 18 & 19 have quotes, but lines 20 & 21 don't?

Reply:
I think it's due to copy and paste from another `docker-compose.yaml`.
Semantically they make no difference, but I'll use the same format for consistency.